llvm/test/Transforms/LoopVectorize/reduction-inloop.ll

```diff
@@ -627,7 +1091,7 @@ for.end:                                          ; preds = %for.body, %entry
   ret float %result.0.lcssa
 }
 
-; Sub we can create a reduction, but not inloop
+; We can create an in-loop reduction for sub-reductions by negating the input
```

Comment:
nit: this comment is no longer correct (the input is not negated, since it just uses `sub` instead)